test/e2e/storage/persistent_volumes-local.go

```diff
@@ -344,11 +347,24 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			kind := schema.GroupKind{Group: "extensions", Kind: "DaemonSet"}
 			framework.WaitForControlledPodsRunning(config.client, config.ns, daemonSetName, kind)
 
-			By("Creating a directory under discovery path")
-			framework.Logf("creating local volume under path %q", volumePath)
-			mkdirCmd := fmt.Sprintf("mkdir %v -m 777", volumePath)
-			err := framework.IssueSSHCommand(mkdirCmd, framework.TestContext.Provider, config.node0)
+			By("Creating a directory for content")
+			framework.Logf("creating local directory at path %q", volumeContentDir)
+			mkdirCmd1 := fmt.Sprintf("mkdir %v -m 777", volumeContentDir)
+			mkdirCmd1Err := framework.IssueSSHCommand(mkdirCmd1, framework.TestContext.Provider, config.node0)
```

Comment:
nit - go idiom would be to move the `err` var declaration here (from L365) and re-assign it for each error catch.